app/assets/javascripts/govuk_publishing_components/components/layout-super-navigation.js

```diff
@@ -0,0 +1,70 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function SuperNavigationToggle ($module) {
+    this.$module = $module
+    this.$button = _setupButton($module)
+    this.$menu = document.getElementById(this.$button.getAttribute('aria-controls'))
+    this.status = this.$button.getAttribute('aria-expanded') === 'true' ? 'open' : 'closed'
+  }
+
+  function _removeClass (classes, classToRemove) {
```

Comment:
We can use the `classList` polyfill and do this natively with `classList.remove`
```
//= require govuk/vendor/polyfills/Element/prototype/classList.js
```

Reply:
Done - and the rest of the JavaScript has been updated to us `classList`.